core/psh/sysexec.c

```diff
@@ -25,6 +27,63 @@ void psh_sysexecinfo(void)
 }
 
 
+static int psh_sysexec_wldargmatch(char *wld, int argc, char **argv, char wldend, int argmaxlen)
+{
+	int len = 0, ii;
+	char *currc = wld, *nextc, *cmdend;
+
+	cmdend = strchr(wld, wldend);
+	currc = wld;
+	for (ii = 0; ii < argc; ii++) {
```

Comment:
Why call iterator `ii` instead of standard `i`?

Reply:
The `ii` iterator style is used in order to easy look for (ctrl+f) occurances of loop variable (it is highly rare to use double letter 'i', so ctrl+f -> `ii` finds you all occurances of that iterator and not just all `i` e.g in word _while_. If it is misleading in any way I can abandon that iterator styling.

Reply:
it's ok to use variable names like that